released/discovery_center/mission_4327/setup_subaccount_cf/README.md

```diff
@@ -13,13 +13,13 @@ The [Terraform](https://www.terraform.io/) script  documented here automates the
 
 Before using this script, make sure you have Terraform downloaded and installed.
 
-- **Install Terraform**: If you haven't already, download and install Terraform from [here](https://www.terraform.io/downloads.html).
+- **Install Terraform**: If you haven't already, download and install Terraform (We currently having an issue when using the Terraform CLI in version 1.6.x, please install Terraform CLI version 1.5.7) from [here](https://www.terraform.io/downloads.html).
```

Comment:
Please reference the getting started readme her <https://github.com/SAP-samples/btp-terraform-samples/blob/main/GET_STARTED.md>